SYCL/Basic/image/image_read.h

```diff
@@ -7,7 +7,7 @@
  *
  * https://www.khronos.org/registry/OpenCL/specs/opencl-1.2.pdf#page=329
  *
- * Confirmed to pass with -fsycl-targets=spir64-unknown-linux-sycldevice
+ * Confirmed to pass with -fsycl-targets=spir64-unknown-linux
```

Comment:
I have no idea what this comment is about. Can we delete it?

Reply:
Should be good to remove, looks quite outdated